### ISSUE TITLE
Ads Wont Play

### ISSUE THREAD

**Issue by yummiesttag** [Focus/youtubequality#8]
I have a issue where if i have the pause video on startup on if a ad on youtube comes up when you press play it plays for a second and then pauses again

**Comment by Focus** [comment]
Does it pause during the ad? If so, there isn't much I can do, youtube api has nothing concerning ads. I am not sure if one can even detect if ads have stopped playing.

**Comment by yummiesttag** [comment]
Yes it pauses the ad and when I press play it immediately repauses it
